src/Microsoft.AspNetCore.Mvc.TagHelpers/CacheTagHelperBase.cs

```diff
@@ -99,6 +99,14 @@ public CacheTagHelperBase(HtmlEncoder htmlEncoder)
         public TimeSpan? ExpiresAfter { get; set; }
 
         /// <summary>
+        /// Gets or sets the default duration, from the time the cache entry was added, when it should be evicted.
+        /// This default duration will only be used if no other expiration criteria gets set for the tag helper.
+        /// The default expiration time is a sliding expiration of 30 seconds.
+        /// </summary>
+        [HtmlAttributeNotBound]
+        public TimeSpan DefaultExpiration { get; set; } = TimeSpan.FromSeconds(30);
```

Comment:
suggest making this public static readonly field - there's no real scenario for changing it, you would set one of the other properties instead